src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -117,10 +173,57 @@ class pt7(Macro):
     
     def run(self, *args, **kwargs):
         pass
+    
+class pt7d1(Macro):
+    """Macro with a list of pair Motor,Float. Default value for last ParamRepeat element.
+    Usages, ex.:
+    pt7d1 [[mot1 1][mot2 3]]
+    pt7d1 [mot1 1 mot2 3]
+    pt7d1 mot1 1 mot2 3
+    Using default value, ex.:
+    pt7d1 [[mot1][mot2 3]] # at any repetition
+    pt7d1 [mot1 1 [mot2]] # only at the last repetition
+    
+    """
+
+    param_def = [
+       [ 'm_p_pair', [ [ 'motor', Type.Motor, None, 'Motor to move'],
+                       [ 'pos',   Type.Float, 2, 'Position to move to'] ],
+         None, 'List of motor/position pairs']
+    ]
+    
+    def run(self, *args, **kwargs):
+        pass
+
+    
+class pt7d2(Macro):
+    """Macro with a list of pair Motor,Float. Default value for both ParamRepeat elements.
+    Usages, ex.:
+    pt7d2 [[mot1 1][mot2 3]]
+    pt7d2 [mot1 1 mot2 3]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.